pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [tool.mypy]
 show_error_codes = true
-disable_error_code = "attr-defined"
+disable_error_code = "attr-defined, name-defined"
```

Comment:
I needed to add `name-defined` here because mypy got more strict about erroring on undefined names from compiled extensions; e.g. `jaxlib.mlir.ir`. A more comprenensive fix here would be to add full typestubs to all jaxlib extensions, but that would not be a trivial undertaking.